### ISSUE TITLE
FAQs

### ISSUE THREAD

**Issue by tonyhb** [tonyhb/tectonic#29]
q: I deleted an item but it's still in my list
a: the model ID was not passed as a parmater to the delete function, therefore we could not figure out which model to remove from the reducer state

**Comment by greghawk** [comment]
@tonyhb So did you drop the redux-boilerplate to do this instead?

**Comment by tonyhb** [comment]
Yeah, it's sorta graphql like but for rest APIs. Got tires of the endless
action/middleware/normalizr/reducer/reselect ceremony when we cant use
graphql.

I'll write up some docs soon then hopefully back to redux ui and the book.

You thinking of using this?

**Comment by greghawk** [comment]
@tonyhb I am.  I have been going down the path with redux/react and seeing the pros and cons.  I was originally looking at what corey house had done with react slingshot but you seem to have a better vetted concept with the boilerplate that separates concerns better.  I also wanted to do some work with falcor and you had mentioned where that may play in.  If you can believe it my company is mired in angular 1.2, .net mvc and other nonsense.  I am trying to put together a boiler/starter that we could use where i could show them the light.  I know angular well but I don't like it at all.

**Comment by tonyhb** [comment]
I hear you. Actually have a semi-start repo for personal projects with
boilerplate golang and UI - you're welcome to use it. Will link here in a
sec.

Sorry to hear about the angular world dude. I hear C# isn't so bad if
that's what you're using though?

**Comment by tonyhb** [comment]
The project is at https://github.com/tonyhb/personal-starter.  It could use some more things like inlining SVGs, https://github.com/styled-components/styled-components, and https://github.com/mbasso/react-decoration though.

The ui is all in ui 😄

**Comment by greghawk** [comment]
C# is great for windows services.  It is and has always been horrible for web. It can do well in enterprise middle-ware situations.  There is a .Net core now that is multi-platform like java but it is in its infancy and we all know that things that are new from microsoft take a long time to shake out.

**Comment by greghawk** [comment]
what made you want to play with golang? More low level without being C?

**Comment by tonyhb** [comment]
Ah that makes sense. Never used C# but totally understand that it wouldnt be cut out for the web.  Ya, you pretty much hit it.  Easy concurrency, good standard library which lets you do most stuff (limited readers, pipes, networking, buffering etc.), which has been useful for a few projects.  It's actually about as good as it gets for web stuff without holding your hands like rails/elixir.  Definitely wanna use elixir next!

What are you working on (both at work and outside)?

**Comment by greghawk** [comment]
Don't have a lot of outside time.  I have two kids and I play in a band.  At work I am working to redesign the entire UX flow.  I work in UX design as well as dev.  I'm a little bit of an oddball that way.

**Comment by greghawk** [comment]
angular is too monolithic and inflexible to do multivariate and performant ui work.  Especially when it is badly written hand me down code.

**Comment by greghawk** [comment]
@tonyhb I just tried to clone the redux-boilerplate and it says i don't have permission?

**Comment by greghawk** [comment]
@tonyhb nevermind i was being a gitshit (DipShit)

**Comment by tonyhb** [comment]
lol no sweat. ah i hear you, family is important as well. lmk if you need anything!  tonyhb@gmail.com :)
